### DISCUSSION TITLE
Scroll bar on proxy options

### DISCUSSION THREAD

**Discussion by elliamorse** [chilli-axe/mpc-autofill#200] [Ideas]
I would love to see a bar that you can use to scroll through proxy options, or a better way to search and skip to a particular number. Currently, it can be difficult to skip to options further down that do not automatically load. For example, if I want three plains, one of numbers 501, 502, and 503, it might take me about 20 seconds to manually scroll back down to where I was. The built-in bar on Chrome can not be used for this since it will only scroll to the bottom for however many of the proxies are loaded. I attached a photo of the bar in Chrome browser that I would love to see present! Love the work, thanks so much for mpcfill!!

**Comment by elliamorse** [comment]
<img width="882" alt="Screenshot 2023-12-26 at 9 08 19 PM" src="https://github.com/chilli-axe/mpc-autofill/assets/143649712/bd5b2a6c-a614-426a-aa39-a37a68dcda39">

**Comment by ndepaola** [comment]
howdy! it looks like you're using the old frontend rather than the new one (currently only hosted at https://mpcautofill.github.io/?server=https://mpcfill.com). there are a couple of changes in the new version which should help you out here:

* the scroll bar works as you expect it to - e.g. dragging the bar halfway through a list of 1000 options will take you to option 500
* you may specify the image version to pre-select when adding cards - e.g. if i want to add four copies of `island` and pre-select version `17eBHjSOwO7uQ9huL0NGhFkXWIYvDCNiv` (this ID is listed in the detailed view modal which pops up when you click the thumbnail), i would type in `4x island@17eBHjSOwO7uQ9huL0NGhFkXWIYvDCNiv`. this is explained in the new syntax guide:
![image](https://github.com/chilli-axe/mpc-autofill/assets/3079166/25e8fa91-f71d-4acd-9182-71d7085b8ec2)

but you're not the first person to ask for the ability to select a specific version by number so i'll add it to the roadmap. this will most likely be an extra form at the top of the version selector modal. i'll keep this discussion open until i get around to adding this (no guarantee on when that'll be), or if someone else jumps in and contributes to the project by adding it. cheers!

**Comment by ndepaola** [comment]
addressed in 303ca76692b7a13d8ec5cf62bd6cebd6ca138f8d, here's how it looks - let me know if this meets your use case:
![image](https://github.com/chilli-axe/mpc-autofill/assets/3079166/3957ee98-d188-4a0d-b097-9c430a28768e)

**Comment by ndepaola** [comment]
closing this as complete

**Comment by elliamorse** [comment]
Thank you so much! This is super exciting, I really appreciate it :)
